lib/utils/export/types.ts

```diff
@@ -0,0 +1,18 @@
+import * as T from '../../types';
+
+export type ExportNote = {
+  content: string;
+  collaboratorEmails: T.TagName[];
+  creationDate: T.SecondsEpoch;
+  id: T.EntityId;
+  markdown?: boolean;
+  modificationDate: T.SecondsEpoch;
+  pinned?: boolean;
+  publicURL?: string;
+  tags: T.TagName[];
+};
+
+export type GroupedExportNotes = {
+  activeNotes: ExportNote[] | [];
```

Comment:
no need for the `| []` here as an empty array is still `ExportNote[]`

Reply:
I was wondering about that. Thx!